theme/typography.ts

```diff
@@ -0,0 +1,59 @@
+import { ColorStates } from "./constants";
+
+interface Measurements {
+  [key: string]: string;
```

Comment:
Can this be typed out better? Enums might be a good approach

Reply:
Something like this?

```
enum Sizes {
  xs = "xs",
  sm = "sm",
  rg = "rg",
  md = "md",
  lg = "lg",
  xl = "xl"
}

enum Headers {
  subheader2 = "subheader2",
  subheader = "subheader",
  header = "header"
}

interface Measurements {
  [Sizes.xs]: string;
  [Sizes.sm]: string;
  [Sizes.rg]: string;
  [Sizes.md]: string;
  [Sizes.lg]: string;
  [Sizes.xl]: string;
  [Headers.subheader2]?: string;
  [Headers.subheader]?: string;
  [Headers.header]?: string;
}
```